### ISSUE TITLE
Response box

### ISSUE THREAD

**Issue by lppr12** [lppr12/Baby-GPT#3]
The response by the bot is not being displayed properly. It has large spaces between consecutive lines. Try fixing it. :)

**Comment by vijayNerdballer** [comment]
by "large spaces" do you mean this shaded portion?
![issue1](https://user-images.githubusercontent.com/79411320/230842118-b1308028-cc15-4631-8598-8c5c0fe95bb0.png)

**Comment by lppr12** [comment]
Yeah. You see the response box appends the string returned by the bot and
it has lot of \n that causes the extra spaces. You can either solve js code
or by tampering with the html and removing extra divs.
:)

**Comment by vijayNerdballer** [comment]
on it! will let you know once I figure it out

**Comment by lppr12** [comment]
Sure 🙂

**Comment by vijayNerdballer** [comment]
hey! check this out, is it satisfactory?
I'm hosting the page on my GitHub in case you want to check it out yourself
![image](https://user-images.githubusercontent.com/79411320/230899312-e17b7697-a558-4a19-8652-d5796ccb67c5.png)

**Comment by lppr12** [comment]
> hey! check this out, is it satisfactory?
> 
> ![image](https://user-images.githubusercontent.com/79411320/230899312-e17b7697-a558-4a19-8652-d5796ccb67c5.png)

Can you ask it to write a small code. I want to see how it indents it.

**Comment by vijayNerdballer** [comment]
> > hey! check this out, is it satisfactory?
> > ![image](https://user-images.githubusercontent.com/79411320/230899312-e17b7697-a558-4a19-8652-d5796ccb67c5.png)
> 
> Can you ask it to write a small code. I want to see how it indents it.

sure
![image](https://user-images.githubusercontent.com/79411320/230905898-a7b59639-9bca-4819-a09a-66e65ff2a6a0.png)

**Comment by vijayNerdballer** [comment]
sure

**Comment by vijayNerdballer** [comment]
here
![image](https://user-images.githubusercontent.com/79411320/230906179-69d42ed2-a2a1-4b46-97bb-048b2caa4ee9.png)

**Comment by vijayNerdballer** [comment]
> See those spaces after line break, if you can decrease those a bit it will be perfect. :)

I see! will hit you up once I'm done

**Comment by vijayNerdballer** [comment]
texted you on baby gpt channel on hackodex's discord! look it up
